app/scripts/controllers/authentication/authentication-controller.ts

```diff
@@ -251,6 +254,9 @@ export default class AuthenticationController extends BaseController<
         accessToken,
       };
     } catch (e) {
+      console.error('Failed to authenticate', e);
+      // Disable Profile Syncing
+      this.messagingSystem.call('UserStorageController:disableProfileSyncing');
```

Comment:
Why are we disabling profile syncing if auth fails? Shouldn't we treat these as 2 actions, do we need to tie them together?

Reply:
OK this is valid. If auth fails (edge case), then we will need to turn off profile syncing.

Reply:
Ticket: https://consensyssoftware.atlassian.net/browse/NOTIFY-653